views/__init__.py

```diff
@@ -1001,7 +998,15 @@ def about_page():
     female_patients=patients_db.patients.find( {'sex':'F'}).count()
     print('female_patients',female_patients,)
     unknown_patients=patients_db.patients.find( {'sex':'U'}).count()
-    return render_template('about.html',total_patients=total_patients)
+    try:
+        version_number = subprocess.check_output(['git', 'describe', '--exact-match'])
```

Comment:
Personally I'm not too sure about using `git` for versioning - mainly because we aren't really using tags... If we want to go down this route, we will need to change the way we commit to the master branch and ensure we regularly create tags. If we want to go down this route, maybe add some info in Contributing.md documenting this (with maybe a few lines of code showing how to create a tag)...

In my opinion, it would be easier to simply have a global version variable that we manually update in line with semantic versioning - i.e. at set periods... 

Alternatively run `git describe --all` to get the 6 character commit hash (e.g. `13a37a0`), which can also be used for versioning (but will be less meaningful for anyone who wants to know, at a glance, which version is the latter version)...

Reply:
The code checks first for a tag. If no tag is found, it gets the date of the last commit on this branch instead, so the webpage will display "Version: Sun Oct 15 17:17:10 2017".

Tags are preferable. Ideally, the release script for phenopolis would include a line that sets a tag. Here is how to do it:-
```git tag``` 			                                                show all tags
```git tag -a v1.0.2 -m "Some release message." ``` 	create a tag
```git push origin v1.0.2```	                                        push tag to repository
@pontikos - could you add this to your release script?

It'd be more orderly to tag it at the time of release, rather than having different contributors setting tags at different times, possibly using different numbering systems.